test/localnet/utils/tests/__snapshots__/getChains.test.mjs.snap

```diff
@@ -124,7 +124,6 @@ Object {
   },
```

Comment:
What is this file for? How is this file generated?

Reply:
This is @canercandan's file, from what I saw its only reason for existence is that a unit test can compare it to another JSON file. The test broke when I fixed the other JSON file and from my point of view this test makes no sense at all, but it's out of scope for my work and @canercandan was unavailable at the time to confirm it.